charts/helm_lib/README.md

```diff
@@ -0,0 +1,847 @@
+# Helm utils template definitions for Deckhouse modules
```

Comment:
"Helm utils template definitions" could be just "Helm library" because there cannot be anything more than reusable definitions AFAIK.

```suggestion
# Helm library for Deckhouse modules
```

Reply:
done